x-pack/plugin/esql/qa/testFixtures/src/main/resources/bucket.csv-spec

```diff
@@ -415,13 +415,69 @@ FROM employees
 | LIMIT 4
 ;
 
- sumK:double   | b1k:double    | b2k:double      
+ sumK:double   | b1k:double    | b2k:double
 49.0           |25000.0        |24000.0
 52.0           |26000.0        |26000.0
 53.0           |27000.0        |26000.0
 56.0           |28000.0        |28000.0
 ;
 
+reuseGroupingFunction#[skip:-8.13.99, reason:BUCKET renamed in 8.14]
+FROM employees
+| STATS sum = 1 + BUCKET(salary, 1000.) BY b1k = BUCKET(salary, 1000.)
+| SORT sum
+| LIMIT 4
+;
+
+ sum:double    | b1k:double
+25001.0        |25000.0
+26001.0        |26000.0
+27001.0        |27000.0
+28001.0        |28000.0
+;
+
+reuseGroupingFunctionWithinAggs#[skip:-8.13.99, reason:BUCKET renamed in 8.14]
```

Comment:
Super nit: we don't skip because bucket was renamed; this wasn't possible with auto bucket, I think.

Reply:
That's true, though we do skip the -8.13.99 tests because BUCKET was renamed. 🤔